config/locales/en.yml

```diff
@@ -531,9 +535,20 @@ en:
           - "No"
           - "Not sure"
         items:
+          - text: "Find out if you can get help if you’re clinically at high risk from coronavirus (Gov.scot)"
+            href: "https://www.gov.scot/publications/covid-shielding/pages/overview/"
+            show_to_nations:
+              - Scotland
           - text: "You might be able to phone or email your local shops to get a food delivery, or get food online."
           - text: 'If you can, ask friends, family or neighbours to help you get food. <a href="https://www.gov.uk/guidance/coronavirus-covid-19-accessing-food-and-essential-supplies" class="govuk-link">Find out how you can pay for your shopping safely.</a>'
           - text: 'If there’s no one to help you get food, prescriptions, or other essentials, find out if you can get help from a volunteer through the <a href="https://volunteering.royalvoluntaryservice.org.uk/nhs-volunteer-responders-portal/isolating" class="govuk-link"> NHS Volunteer Responders programme.</a>'
+            show_to_nations: 
+            - England
+            - Wales
+            - Northern Ireland
+          - text: 'If there’s no one to help you get food, prescriptions, or other essentials, <a href="https://www.gov.scot/publications/coronavirus-covid-19-help-for-vulnerable-people/" class="govuk-link"> call the coronavirus helpline for Scotland </a>' or find out if you can get help from a volunteer through the <a href="https://volunteering.royalvoluntaryservice.org.uk/nhs-volunteer-responders-portal/isolating" class="govuk-link"> NHS Volunteer Responders programme.</a>'
```

Comment:
```suggestion
          - text: 'If there’s no one to help you get food, prescriptions, or other essentials, <a href="https://www.gov.scot/publications/coronavirus-covid-19-help-for-vulnerable-people/" class="govuk-link"> call the coronavirus helpline for Scotland </a> or find out if you can get help from a volunteer through the <a href="https://volunteering.royalvoluntaryservice.org.uk/nhs-volunteer-responders-portal/isolating" class="govuk-link"> NHS Volunteer Responders programme.</a>'
```

Reply:
yep that was it, just committed and squahsed the change